packages/venia-concept/src/components/SignIn/signIn.js

```diff
@@ -1,12 +1,18 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import get from 'lodash/get';
```

Comment:
Right now we did not define lodash as a dependency in our package.json, but I think that `get` function from lodash is extremely useful. @zetlen, @jimbo Could we discuss adding lodash.get https://github.com/lodash/lodash/tree/4.4.2-npm-packages/lodash.get to the project?